src/interpreter/mod.rs

```diff
@@ -393,7 +393,7 @@ impl Interpreter {
     }
 
     /// Interprets the whole currently set program and returns the
-    /// used/unused values after the last statment.
+    /// used/unused values after the last statement.
```

Comment:
Thank you for fixing typos, but this inflates the PR scope. Would be better to not change code you don't need to change, even if there are typos 😉

Reply:
Does this apply to the missing periods too?

Reply:
You are testing me... 🐱